tests/Integration/tests/index/expected/objects.inv.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Why is this file part of the integration tests? It is not checked anywhere?

Reply:
yes it is. It is the expected output of the file